client/account-status-settings/index.js

```diff
@@ -81,7 +81,7 @@ const renderAccountStatusDescription = ( accountStatus ) => {
 		);
 	} else if ( 'rejected.terms_of_service' === status ) {
 		description = __(
-			'This account has been rejected due to a Terms of Service violation.',
+			'This account has been rejected due to a Terms of service violation.',
```

Comment:
I'm not sure about "Terms of Service" vs "Terms of service" or everything lowercase. Let's ask Jarek or David about this.

Reply:
Somehow I ended up merging another branch inside of the one I was working. I've reverted the merge now

Reply:
I've asked David, the issue author about this and "Multi-Currency": https://github.com/Automattic/woocommerce-payments/issues/1323#issuecomment-933759751